### PR TITLE
Fixes #22697 - improve the invocation form

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: app/views/job_invocations/_form.html.erb [hunk 1]
```diff
@@ -90,17 +90,17 @@
     <% end %>
 
     <div class="advanced hidden">
-      <%= password_f f, :password, :placeholder => '*****', :label => _('Password') %>
-      <%= password_f f, :key_passphrase, :placeholder => '*****', :label => _('Private key passphrase') %>
+      <%= password_f f, :password, :placeholder => '*****', :label => _('Password'), :label_help => N_('Password is stored encrypted in DB until the job finishes. For future or recurring executions, it is removed after last such execution.') %>
+      <%= password_f f, :key_passphrase, :placeholder => '*****', :label => _('Private key passphrase'), :label_help => N_('Key passhprase is only applicable for SSH provider. Other provider ignores this field. <br> Passphrase is stored encrypted in DB until the job finishes. For future or recurring executions, it is removed after last such execution.') %>
```
Comment (dLobatog) [inline app/views/job_invocations/_form.html.erb]: `Other providers ignore this field`
`it is removed after the last execution`

File: app/views/job_invocations/_form.html.erb [hunk 2]
```diff
@@ -90,17 +90,17 @@
     <% end %>
 
     <div class="advanced hidden">
-      <%= password_f f, :password, :placeholder => '*****', :label => _('Password') %>
-      <%= password_f f, :key_passphrase, :placeholder => '*****', :label => _('Private key passphrase') %>
+      <%= password_f f, :password, :placeholder => '*****', :label => _('Password'), :label_help => N_('Password is stored encrypted in DB until the job finishes. For future or recurring executions, it is removed after last such execution.') %>
```
Comment (dLobatog) [inline app/views/job_invocations/_form.html.erb]: ` it is removed after the last execution`

**Comment by ares** [comment]
Thanks @dLobatog, updated the strings as suggested.

**Comment by iNecas** [comment]
Thanks @ares and @dLobatog

**Pull request merged**